e2e/test/scenarios/models/model-actions.cy.spec.js

```diff
@@ -100,7 +100,9 @@ describe(
 
       cy.findByRole("link", { name: "New action" }).click();
       fillActionQuery("DELETE FROM orders WHERE id = {{ id }}");
-      fieldSettings().findByText("Number").click();
+      cy.findByRole("radiogroup", { name: "Field type" })
```

Comment:
@WiNloSt we could, but clicking `RadioInput` in Cypress is tricky because it's visually hidden. We keep the HTML input component there for a11y reasons, but it's actually a `label` that is clicked. If we try to do `cy.findByRole("radio").click()`, Cypress would complaint the element is invisible

Reply:
Thanks for the explanation.